test/e2e/logs_test.go

```diff
@@ -0,0 +1,49 @@
+package e2e
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// testNodeLogs ensures that all required log files were created, and copies them to the test's artifact directory
+// It also tests that 'oc adm node-logs' works with the nodes created by WMCO.
+func testNodeLogs(t *testing.T) {
+	// All these paths are relative to /var/log/
+	logFiles := []string{
+		"kube-proxy/kube-proxy.exe.INFO",
+		"kube-proxy/kube-proxy.exe.ERROR",
+		"kube-proxy/kube-proxy.exe.WARNING",
+		"hybrid-overlay/hybrid-overlay.log",
+		"kubelet/kubelet.log",
+	}
+
+	nodeArtifacts := filepath.Join(os.Getenv("ARTIFACT_DIR"), "nodes")
+	for _, node := range gc.nodes {
+		nodeDir := filepath.Join(nodeArtifacts, node.Name)
+		for _, file := range logFiles {
```

Comment:
This should follow a best effort mode to try and get all the logs and not bail on the first error.